src/Raven.Server/Documents/AI/ChatCompletionClient.cs

```diff
@@ -530,25 +445,35 @@ private static bool TryParseResetTime(string input, out TimeSpan time)
         return true;
     }
 
+    public static string GetSchema(string schema, string sampleObject, bool forParameters = false)
```

Comment:
I don't like `forParameters`, let's split that into 
`GetSchemaForTool` and `GetSchemaForRequest`